pkgs/os-specific/linux/autosuspend/disable-cov.patch

```diff
@@ -0,0 +1,19 @@
+Common subdirectories: source.orig/.github and source/.github
+Common subdirectories: source.orig/data and source/data
+Common subdirectories: source.orig/doc and source/doc
+diff -u source.orig/setup.cfg source/setup.cfg
+--- source.orig/setup.cfg	1980-01-02 00:00:00.000000000 +0100
++++ source/setup.cfg	2021-10-05 10:34:40.837438296 +0200
+@@ -55,8 +55,8 @@
+ filterwarnings =
+     ignore::DeprecationWarning
+     default::DeprecationWarning:autosuspend
+-addopts =
+-    --cov-config=setup.cfg
++#addopts =
++#    --cov-config=setup.cfg
```

Comment:
Patches are bridle and easily break in the python world. Using
```
substituteInPlace setup.cfg \
  --replace '--cov-config=setup.cfg' ""
```
is more robust and causes less merge conflicts.

Also please don't comment things in patches, just drop them.